ocaml/parsing/jane_syntax_parsing.ml

```diff
@@ -7,23 +7,29 @@
       of the syntax indicated by the attribute.
     2. As a pair of an extension node and an AST item that serves as the "body".
        Here, the "pair" is embedded as a pair-like construct in the relevant AST
-       category, e.g. [include sig [%%extension.EXTNAME];; BODY end] for
+       category, e.g. [include sig [%jane.ERASABILITY.EXTNAME];; BODY end] for
        signature items.
 
-    In particular, for an extension named [EXTNAME] (i.e., one
-    that is enabled by [-extension EXTNAME] on the command line), the attribute
-    (if used) must be [[@jane.EXTNAME]], and the extension (if used) must be
-    [[%jane.EXTNAME]]. For built-in syntax, we use [_builtin]
-    instead of an extension name.
+    In particular, for an language extension named [EXTNAME] (i.e., one that is
+    enabled by [-extension EXTNAME] on the command line), the attribute (if
+    used) must be [[@jane.ERASABILITY.EXTNAME]], and the extension node (if
+    used) must be [[%jane.ERASABILITY.EXTNAME]]. For built-in syntax, we use
+    [_builtin] instead of an language extension name.
+
+    The [ERASABILITY] component indicates to tools, like ppxlib, whether the
+    attribute is erasable or not. See the documentation of [Erasability] for
+    more information on how tools make use of this information.
 
     In the below example, we use attributes an examples, but it applies equally
     to extensions. We also provide utilities for further desugaring similar
     applications where the embeddings have the longer form
-    [[@jane.FEATNAME.ID1.ID2.….IDn]] (with the outermost one being the [n = 0]
-    case), as these might be used inside the [EXPR]. (For example, within the
-    outermost [[@jane.comprehensions]] term for list and array comprehensions,
-    we can also use [[@jane.comprehensions.list]],
-    [[@jane.comprehensions.array]], [[@jane.comprehensions.for.in]], etc.).
+    [[@jane.ERASABILITY.FEATNAME.ID1.ID2.….IDn]] (with the outermost one being the
+    [n = 0] case), as these might be used inside the [EXPR]. (For example,
+    within the outermost [[@jane.ERASABILITY.comprehensions]] term for list and
```

Comment:
Comprehensions are `non_erasable`; all the examples using them should make that explicit

Reply:
Done.